File: plusmall-operator-web/src/main/webapp/js/service/sellerService.js
app.service('operatorSellerService',function ($http) {

    //查找商家
    this.search=function (pageNum, pageSize,searchEntity) {
        return $http.post('../seller/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    }

    //查找单个商家信息
    this.findOne=function (strId) {
        return $http.get('../seller/findOne.do?strId='+strId);
    }

    //更新商家信息
    this.update=function (seller) {
        return $http.post('../seller/update.do',seller);
    }

    //更新商家状态
    this.updateStatus=function (strId,status) {
        return $http.get('../seller/updateStatus.do?strId='+strId+'&status='+status);
    }
});

File: plusmall-operator-interface/src/main/java/com/plusmall/operator/SellerService.java
package com.plusmall.operator;

import com.plusmall.commons.PageResult;
import com.plusmall.model.TbSeller;

/**
 * @Description:
 */
public interface SellerService {
	public PageResult search(int pageNum, int pageSize, TbSeller seller);
	public void delete(Long[] ids) throws NullPointerException;
	public void add(TbSeller seller) throws NullPointerException;
	public TbSeller findOne(String sellerId);
	public void udpate(TbSeller seller) throws NullPointerException;
	public void updateStatus(String strId,String status);
}
